main.py

```diff
@@ -1,7 +1,27 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    Genre.objects.create(name="Western")
```

Comment:
You are making lots of code repeating. It would be better to create lists of genres' names and actors' data and create Genres and Actors instances using `for` loop